grpc-java/grpc-servlet-websocket-jakarta/src/main/java/io/grpc/servlet/web/websocket/MultiplexedWebsocketStreamImpl.java

```diff
@@ -76,7 +76,7 @@ private void writeMetadataToStream(Metadata headers, boolean closeBitSet) {
             try {
                 websocketSession.getBasicRemote().sendBinary(message);
             } catch (IOException e) {
-                throw Status.fromThrowable(e).asRuntimeException();
+                // ignore IOExceptions; they are likely due to unpredictable client behavior
```

Comment:
I agree that this should not throw, and onError should be called automatically, so we shouldn't need to do specific handling here for that case. Can you adjust the comment to reflect that, so it is clear that this case is handled, rather than just being ignored?